### DISCUSSION TITLE
Samsung AC090HCAFKH Works perfectly! Not sure if it's NASA or NON-NASA though...

### DISCUSSION THREAD

**Discussion by tech2urdoor** [omerfaruk-aran/esphome_samsung_hvac_bus#198] [General]
A huge thank you to everyone who is involved with this project! I have a ducted AC system (AC090HCAFKH). I had a MIM-H02 Wifi Kit attached to the controller. The app that used to work with it was pulled from the app store by Samsung and they had lost all interest in supporting this wifi kit: https://community.smartthings.com/t/mim-h02-support/245691/7 I was nearly going to buy the upgraded MIM-H04 unit so I could control it once again but didn't want to give Samsung anymore money for something that would most likely lose support in the future.

I opened up the wifi kit cover and all the wires were there ready to go. I disconnected them from the board of the kit and attached the hardware (Atom stack) that was suggested and programmed it all up, it's working perfectly! I even managed to get the chip inside the old wifi kit case and put the cover back on. It's so clean! and I can control it all through home assistant!! I'm so happy.

Just a small note though, In the readme and example it talked about NASA and Non-NASA. My unit didn't seem to fit either category. It discovered an Outdoor, Indoor and Other...

I was always of the belief that it was a Non-NASA but according to the example.yaml if it was Non-NASA then it shouldn't have discovered an Outdoor device let alone an Other device...

I'm in Australia if that means anything. There is an outdoor unit and a control panel inside that controls what is essentially one unit with 
4 different zones that are only controllable with a separate controller called a zonemaster. The wifi kit with the Samsung could never control zones. My next project is to see if I can somehow wire an esp to the controller panel now. 

I just wanted to thank everyone involved and share that it works with a model that I didn't see listed as compatible.

**Comment by omerfaruk-aran** [comment]
Thank you for sharing your success with integrating the AC system! 😊

Regarding the NASA/Non-NASA question, based on the outdoor unit model "AC090HCAFKH" that you've provided, it seems like your unit falls under the Non-NASA category. This can be confirmed by referring to the model naming convention (as seen in the attached documentation). Specifically, in the product type identifier, the letter "C" indicates a Non-NASA outdoor unit.

If there are any additional questions or if you'd like more information, feel free to ask! Your setup sounds great, and I’m glad to hear everything is working smoothly with Home Assistant!

![image](https://github.com/user-attachments/assets/8733f0d5-68a8-4c20-a594-a83a0025e323)

**Comment by tech2urdoor** [comment]
Thank you Omar! Because I have an outdoor device listed. Do you think I should edit the yaml and see if I can get an outdoor temperature reading from the outdoor unit? Also, any idea what the 'other' device would be? I'm just asking because if I recall according to the info in the example.yaml if it's a non-nasa device it should only come up with indoor devices being discovered. It's a non-issue really, I'm just curious but I certainly don't want to risk breaking anything when it all works perfectly right now. Cheers!

**Comment by Jkirjo** [comment]
That device seen outside temp could be nice addition. Also possible defrosting cycles could be one somehow noticed and visualize in UI. It is beyond of my skills, but someone perhaps could add them also.
